app/assets/stylesheets/application.scss

```diff
@@ -370,3 +370,78 @@ td > .btn:not(:last-of-type) {
 // .userway_buttons_wrapper {
 //   margin-bottom: 5rem;
 // }
+
+.btn-dark-orange {
```

Comment:
Don't make your own button class, use btn-primary. If you insist, use scss mix function to mix primary to darken it. Do not hardcode magic colors. I'm not sure where do we even need a darker color